rules/windows/impact_deleting_backup_catalogs_with_wbadmin.toml

```diff
@@ -13,7 +13,7 @@ Identifies use of the wbadmin.exe to delete the backup catalog. Ransomware and o
 system recovery.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*", "logs-system.*"]
```

Comment:
Should we update metadata integration?

Reply:
Also, should this be `"logs-system.security*"`?

Reply:
++ replaced `logs-system.*` by `logs-system.security*` to be more specific (4688 events) and added system to integreations tag for all rules in this PR.